client/hosting-overview/components/quick-actions-card.tsx

```diff
@@ -33,7 +34,7 @@ const QuickActionsCard: FC = () => {
 	const translate = useTranslate();
 
 	return (
-		<Card className="hosting-overview__card">
+		<Card className={ classNames( 'hosting-overview__card', 'hosting-overview__quick-actions' ) }>
```

Comment:
We don't need `classnames` for this:
```suggestion
		<Card className="hosting-overview__card hosting-overview__quick-actions">
```